doc/configuring.rst

```diff
@@ -436,3 +436,68 @@ If executing the build step depends on more than one environment variable, use `
 For example, ``if_env_set="SPECIAL_TOOL_PATH && ADDITIONAL_SOURCES_ROOT"`` step will be executed only
 in case of both `$SPECIAL_TOOL_PATH` and `$ADDITIONAL_SOURCES_ROOT` environment variables set to some values.
 If any of them is missing or not set in current environment, the step will be excluded from current run.
+
+
+Conditional steps
+---------------------
+
+Conditional step - :class:`Step`, that has ``if_succeeded`` and/or ``if_failed`` parameters with other steps assigned.
```

Comment:
They do not use dashes like this in English.
"Conditional step is a 'Step' object, that has..."